quiz.html

```diff
@@ -11,33 +11,29 @@
 		</header>
 		
 		<div class="wrapper">
+			<div class="quiz-page">
+				<h1>Skin Type Quiz</h1>

```

Comment:
Included a header for the page in this version, whereas I didn't add it in version 1.